setup.py

```diff
@@ -61,7 +61,7 @@ def run(self):
 INSTALL_REQUIRES = ['setuptools',
                     'cryptography>=2.3',
                     'pynacl',
-                    'byteStringSplitter',
+                    'bytestring-splitter',
```

Comment:
It's necessary to update the Pipfile and Pipfile.lock

Reply:
Right - Working on this.

Reply:
:white_check_mark: